cloudformation/membership-attribute-service.json

```diff
@@ -177,6 +177,16 @@
                 "Resource": "arn:aws:dynamodb:*:*:table/friendly-tailor-PROD"
               },
               {
+                "Effect":"Allow",
+                "Action":["sns:Publish"],
+                "Resource":{"Ref":"TopicGiraffe"}
+              },
+              {
+                "Effect":"Allow",
+                "Action":["sns:Publish"],
+                "Resource":{"Ref":"TopicGiraffeUAT"}
```

Comment:
It would be nice if this was parameterised so that only a production instance could talk to the production topic.